notebooks/en/advanced_rag.ipynb

```diff
@@ -6,7 +6,7 @@
     "id": "hUCaGdAj9-9F"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        chunk_size=1000,  # The maximum number of characters in a chunk: we selected this value arbitrarily.
</pre><p>Let's not add any dots at the end for consistency across the cookbook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Thanks. I have removed all dots from comments.